src/components/Routers.js

```diff
@@ -41,14 +41,32 @@ export default function Routers() {
                   }}
                 />
               </Route>
-              {appContext.routes.map(
-                ({ linkTo, exact = false, component, disabled }, idx) => (
+              {appContext.routes.map((route, idx) =>
+                route?.children ? (
+                  route.children.map((child, i) => (
+                    <PrivateRoute
+                      path={child?.linkTo}
+                      component={
+                        child?.disabled ? Unauthorized : child?.component
+                      }
+                      exact={child?.exact}
+                      getScrollContainerRef={
+                        child?.linkTo === '/verify'
+                          ? () => refContainer.current
+                          : undefined
+                      }
+                      key={`route_${i}`}
```

Comment:
Is using index to set `key` values good practice?

Reply:
Sure. However, using index values to generate unique keys could negatively impact performance and may cause issues with component state according to React [documentation](https://reactjs.org/docs/lists-and-keys.html).